ddtrace/contrib/internal/openai/_endpoint_hooks.py

```diff
@@ -750,7 +750,8 @@ class _ResponseHook(_BaseCompletionHook):
     def _record_response(self, pin, integration, span, args, kwargs, resp, error):
         resp = super()._record_response(pin, integration, span, args, kwargs, resp, error)
         if kwargs.get("stream") and error is None:
```

Comment:
Same here, see https://github.com/DataDog/dd-trace-py/pull/13441 for more context @lievan